src/pages/api/conversation.ts

```diff
@@ -185,14 +185,19 @@ export function convertChatToDBMessage(
       .map((content) => content.image_url?.url || '')
   }
 
+  // Ensure contexts is an array before calling map
```

Comment:
This was a long-standing bug, appears whenever `getTopContexts()` fails to return docs. Thanks for this.